.github/workflows/gating.yaml

```diff
@@ -39,7 +39,8 @@ jobs:
       run: |
         pip3 install --upgrade pip
         pip3 install --upgrade setuptools
-        pip3 install --upgrade coveralls==3.2.0
+        pip3 install --upgrade coverage[toml]
```

Comment:
**commit message**

> Note: coveralls is an abandonware now, hasn't been touched
for a couple of years

A reference for such a strong statement would be nice: e.g. this discussion: https://github.com/TheKevJames/coveralls-python/issues/393